Project_Starting_Point/Server/Server.cpp

```diff
@@ -17,8 +20,69 @@ StorageTypes RxData[7];
 void UpdateData(unsigned int, float);
 float CalcAvg(unsigned int);
 
+class LatencyTimer {
+	LARGE_INTEGER frequency{ 0 };
+	LARGE_INTEGER start_t{ 0 }, end_t{ 0 }, elapsedTime{ 0 };
+	vector<pair<string, LONGLONG>> latencyList;
+	
+public:
+	LatencyTimer() : frequency{ 0 }, start_t{ 0 }, end_t{ 0 }, elapsedTime{ 0 } {};
+
+	void start() {
+		QueryPerformanceFrequency(&frequency);
+		QueryPerformanceCounter(&start_t);
+	}
+
+	void end(string functionName = "N/A") {
```

Comment:
If we use 2 classes, this brings up an interesting decision about when this conversion occurs. I feel like it can occur right as we are writing to disk. So maybe in the `LatencyRecorder` class? What do you think?